Add entries for Scourge of the Past

LenartBucar opened the pull request.

LenartBucar committed:
Add entries for Scourge of the Past

Mego commented on src/squares.ts:
```diff
@@ -80,6 +80,33 @@ const raids: { [key: string]: string[] } = {
     "Depositing motes at the wrong relay",
     "Gambit team has no void weapons",
   ],
+  "sotp": [
```
Change to `"Scourge of the Past"` please - I'm using full raid names as the keys, since that's what appears in the raid selection list

LenartBucar replied:
Fixed

LenartBucar committed:
Update squares.ts

Change the name of the raid from the acronym to the full name. 
Reword one of the entries to say `Boss Phase I` instead of just `next encounter`, for clarification.
